swagger.yaml

```diff
@@ -3288,6 +3321,8 @@ definitions:
         type: boolean
       org_seat_limit_reached:
         type: boolean
+      python_performance_first_tier:
```

Comment:
What's all the new stuff for, or why is it here?

Reply:
Good catch. Maybe someone updated and didn't push their changes?